RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -100,6 +100,10 @@ boolean auto_haveMonkeyPaw()
 
 boolean auto_makeMonkeyPawWish(effect wish)
```

Comment:
Nice to have: add check for having paw.

Not currently an issue since you check before calling this function. More to help prevent potential issues from future PRs

Reply:
Done.